Common/Data/Consolidators/DataConsolidator.cs

```diff
@@ -58,7 +58,7 @@ public void Update(IBaseData data)
         /// </summary>
         public IBaseData Consolidated
         {
-            get; private set;
+            get; protected set;
```

Comment:
It could be, but there are other abstract consolidators doing the same (making `Consolidated` protected) like BaseTimelessConsolidator https://github.com/QuantConnect/Lean/blob/master/Common/Data/Consolidators/BaseTimelessConsolidator.cs#L54